RecoMuon/MuonSeedGenerator/python/ancientMuonSeed_cfi.py

```diff
@@ -3,18 +3,15 @@
 from RecoMuon.MuonSeedGenerator.ptSeedParameterization_cfi import *
 from RecoMuon.MuonSeedGenerator.MuonSeedPtScale_cfi import *
 
-# module standAloneMuonSeeds = MuonSeedGenerator {
-ancientMuonSeed = cms.EDProducer("MuonSeedGenerator",
+import RecoMuon.MuonSeedGenerator.produceMuons_cfi as _mod
+ancientMuonSeed = _mod.produceMuons.clone(
                                  ptSeedParameterization,
                                  dphiScale,
                                  beamSpotTag = cms.InputTag("offlineBeamSpot"),
                                  scaleDT = cms.bool(True),
                                  CSCRecSegmentLabel = cms.InputTag("cscSegments"),
                                  DTRecSegmentLabel = cms.InputTag("dt4DSegments"),
                                  ME0RecSegmentLabel = cms.InputTag("me0Segments"),
-                                 EnableDTMeasurement = cms.bool(True),
```

Comment:
The issue here is that the MuonSeedGenerator has  `desc.setAllowAnything()` in its `fillDescriptions` method.
As such, only those three parameters which are removed here from the explicit config are defined, while there is no validation for all other ones. 
For the moment I'd keep the explicit definition here (taking also into account that `EnableME0Measurement` is modified twice in the modifier's below), just removing the type specification.
Eventually, all parameters should be defined and validated inside the `fillDescriptions` method. of MuonSeedGenerator